crates/repl/src/session.rs

```diff
@@ -19,7 +19,7 @@ use runtimelib::{
     ExecuteRequest, InterruptRequest, JupyterMessage, JupyterMessageContent, ShutdownRequest,
 };
 use settings::Settings as _;
-use std::{ops::Range, sync::Arc, time::Duration};
+use std::{env::temp_dir, ops::Range, path::PathBuf, sync::Arc, time::Duration};
```

Comment:
Note: everywhere else we use `paths::temp_dir` which seems to be `~/.cache/zed/`.
Does not matter much, but now it's another place Zed writes to, gets fragmented a bit.